tests/btrfs-progs/install.pm

```diff
@@ -48,12 +49,17 @@ sub run {
     assert_script_run('git clone -q --depth 1 ' . GIT_URL, timeout => 360);
     assert_script_run 'cd btrfs-progs';
     assert_script_run './autogen.sh';
-    assert_script_run './configure --disable-documentation --disable-convert --disable-zstd \
---disable-programs --disable-shared --disable-static --disable-python';
-    assert_script_run 'make testsuite', timeout => 300;
+    assert_script_run './configure --disable-documentation --disable-zstd --disable-python';
+    assert_script_run 'make', timeout => 600;
+    assert_script_run 'make testsuite';
```

Comment:
Or even better you can create bug for btrfs-progs and SLE12-SP5 to ask for update, if you found that version is old.

Reply:
> Why is compilation of btrfs-progs needed during the test? It should be as package and part of QA:Head repo.

Because the package lack of test-suite of btrfs-progs, and only take a few build time. But I'm considering that move btrfs-progs building to OBS in future.

Reply:
> Or even better you can create bug for btrfs-progs and SLE12-SP5 to ask for update, if you found that version is old.

I raised this bug
https://bugzilla.suse.com/show_bug.cgi?id=1140010

Reply:
Greate, thank you.

Reply:
> Because the package lack of test-suite of btrfs-progs, and only take a few build time. But I'm considering that move btrfs-progs building to OBS in future.
Fair enough, thank you.